x/ibc/04-channel/keeper/keeper_test.go

```diff
@@ -45,7 +45,7 @@ const (
 	testChannelOrder   = exported.ORDERED
 	testChannelVersion = "1.0"
 
-	testHeight = 10
+	testHeight = 1
```

Comment:
Quick fix that will solve failing tests on base `ibc-alpha` branch

Reply:
Hmm why? Is there some underlying issue we need to fix?

Reply:
Yes this is a common issue. Back when I added `Height` to ConsensusState I just went through the tests and added a `height` to the consensus states to make everything pass, usually just giving it a random height (10) sufficed. But sometimes if it is getting queried as part of the test, the height needs to match what the non-test code is actually storing. In these tests, we were storing consensus state at height 1 in the non-test code, so i changed the code to fix it

Reply:
Ah I see, that's alright then.